src/components/wallet/methods/depositAsset.ts

```diff
@@ -18,7 +18,12 @@ export default async function depositAsset(
     // If the asset exists in the wallet UI, there exists a record in this.assets
     // however, the homeDomain may not be defined if a trustline for that asset
     // wasn't added in the same session
```

Comment:
Lets remove this comment since it no longer applies right, there may not be a key for the asset within this.assets?